package.json

```diff
@@ -24,7 +24,8 @@
   },
   "icon": "icon.png",
   "engines": {
-    "node": "*"
+    "node": ">=16.0",
```

Comment:
This requirement has implications for extension code. If we enforce this requirement, it should be documented somewhere in the doc/readme file?

Reply:
Extension already has these requirements https://github.com/ansible/vscode-ansible/blob/main/package.json#L334-L339 and I doubt that @yaegassy coc.nvim one would have a problem with that.

Reply:
Per https://github.com/ansible/ansible-language-server/pull/17#discussion_r709585456 and https://github.com/ansible/ansible-language-server/pull/18#issuecomment-920673841, I'm against setting the minimum version so high.
Ideally, it should be 12 but I can agree with a compromise of 14. This project is used as a library and having tight constraints is harmful in such cases.

Reply:
I made it 12, it should be fine for now. We will bump it once if needed.